src/Microsoft.Data.Analysis/strings.resx

```diff
@@ -153,6 +153,9 @@
   <data name="MismatchedColumnLengths" xml:space="preserve">
     <value>Column lengths are mismatched</value>
   </data>
+  <data name="MismatchedColumnValueType" xml:space="preserve">
+    <value>Expected column to hold values of type {0}</value>
```

Comment:
I think this message could use some work.

```C#
PrimitiveDataFrameColumn<int> column = new PrimitiveDataFrameColumn<int>("Int", Enumerable.Range(0, 10));
column.Add(5.5, inPlace: true);
```

Throwing a message saying:

`Expected column to hold values of type System.Single`

might not make sense to users.

Reply:
Your example would use Strings.MismatchedValueType (`Expected value to be of type System.Single`) :). `Strings.MismatchedColumnValueType` is used when you do `column.Add(floatColumn, inPlace: true)`

Reply:
makes sense.